{
    "title" : "Error in Coveo Diagnostic Page",
    "uri" : "http://answers.coveo.com/questions/6162/error-in-coveo-diagnostic-page.html",
    "printableUri" : "https://answers.coveo.com/questions/6162/error-in-coveo-diagnostic-page.html",
    "clickUri" : "https://answers.coveo.com/questions/6162/error-in-coveo-diagnostic-page.html",
    "uniqueId" : "42.4247$http://answers.coveo.com/questions/6162/error-in-coveo-diagnostic-page.html",
    "excerpt" : "As advised coveo, i have updated Coveo for Sitecore to 72 3.0 (1084). ... Now im getting below error in diagnostic page. ... Also im not able to do indexing due to this error. ... Please help me to...",
    "firstSentences" : null,
    "summary" : null,
    "flags" : "HasHtmlVersion",
    "hasHtmlVersion" : true,
    "hasMobileHtmlVersion" : false,
    "score" : 1112,
    "percentScore" : 75.03586,
    "rankingInfo" : null,
    "rating" : 3.0,
    "isTopResult" : false,
    "isRecommendation" : false,
    "titleHighlights" : [ ],
    "firstSentencesHighlights" : [ ],
    "excerptHighlights" : [ ],
    "printableUriHighlights" : [ ],
    "summaryHighlights" : [ ],
    "parentResult" : null,
    "childResults" : [ ],
    "totalNumberOfChildResults" : 0,
    "raw" : {
      "systitle" : "Error in Coveo Diagnostic Page",
      "sysauthor" : "rajkumar",
      "sysurihash" : "UKp8uwwU3eñT8KLð",
      "urihash" : "UKp8uwwU3eñT8KLð",
      "numberofanswers" : 1,
      "sysuri" : "https://answers.coveo.com/questions/6162/error-in-coveo-diagnostic-page.html",
      "sysprintableuri" : "https://answers.coveo.com/questions/6162/error-in-coveo-diagnostic-page.html",
      "sitemappriority" : 0.1,
      "systransactionid" : 160354,
      "questionaskeddatestring" : "·\n            Sep 28, 2015 at 10:21 AM ·",
      "tags" : [ "diagnostic", "page" ],
      "sysconcepts" : "coveo ; Sitecore ; upgrade ; 0Update ; Release ; Hide details ; gone ; andSitecore8 ; StepsspecifictoSitecore7 ; configuration",
      "concepts" : "coveo ; Sitecore ; upgrade ; 0Update ; Release ; Hide details ; gone ; andSitecore8 ; StepsspecifictoSitecore7 ; configuration",
      "printableuri" : "https://answers.coveo.com/questions/6162/error-in-coveo-diagnostic-page.html",
      "documenttype" : "WebPage",
      "sysindexeddate" : 1517942307000,
      "permanentid" : "d39b44a832c846fffc946bf007407aa41be59761b4466b831a62f7445f45",
      "syslanguage" : [ "English" ],
      "transactionid" : 160354,
      "title" : "Error in Coveo Diagnostic Page",
      "date" : 1443398400000,
      "objecttype" : "Answers",
      "sitemapchangefrequency" : "Monthly",
      "answerauthor" : [ "vince" ],
      "hasanswer" : "true",
      "audience" : [ "Administrator", "Developer" ],
      "hasacceptedanswer" : "false",
      "sourcetype" : "Sitemap",
      "sysconnectortype" : "SitemapCrawler",
      "rowid" : 5447819,
      "numberofcomments" : 0,
      "size" : 72652,
      "sysdocumenttype" : "WebPage",
      "questionvotes" : 0,
      "clickableuri" : "https://answers.coveo.com/questions/6162/error-in-coveo-diagnostic-page.html",
      "syssource" : "AnswersCloud",
      "orderingid" : 212384744911578118,
      "acceptedanswers" : 0,
      "syssize" : 72652,
      "sysdate" : 1443398400000,
      "author" : "rajkumar",
      "source" : "AnswersCloud",
      "questionbodyhtml" : "<div class=\"question-body\">\n                <p>Hi Team,</p> \n<p>I have installed \"Coveo for Sitecore 72 3.0 (1026)\" and i have Coveo Enterprise Search 7.0 x64 (7711) installed in my server. As advised coveo, i have updated Coveo for Sitecore to 72 3.0 (1084). Now im getting below error in diagnostic page. Also im not able to do indexing due to this error. Please help me to resolve this error.</p> \n<pre><code> An error occurred while loading the configuration. (Hide details)\n Coveo.Framework.Exceptions.CoveoIndexConfigurationException: An error has occured when creating Coveo Index Configuration in the StateVerifier ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NullReferenceException: Object reference not set to an instance of an object.\n    at Sitecore.ContentSearch.ProviderIndexConfiguration.CreateComputedIndexfield(String fieldName, XmlNode configNode)\n    at Sitecore.ContentSearch.ProviderIndexConfiguration.AddComputedIndexField(XmlNode configNode)\n    --- End of inner exception stack trace ---\n    at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)\n    at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)\n    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)\n    at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)\n    at Sitecore.Configuration.Factory.AssignProperties(Object obj, Object[] properties)\n    at Sitecore.Configuration.Factory.AssignProperties(XmlNode configNode, String[] parameters, Object obj, Boolean assert, Boolean deferred, IFactoryHelper helper)\n    at Sitecore.Configuration.Factory.CreateObject(XmlNode configNode, String[] parameters, Boolean assert, IFactoryHelper helper)\n    at Sitecore.Configuration.Factory.CreateObject[T](XmlNode configNode)\n    at Coveo.SearchProvider.Applications.StateVerifier..ctor()\n    --- End of inner exception stack trace ---\n    at Coveo.SearchProvider.Applications.StateVerifier..ctor()\n    at Coveo.SearchProvider.Applications.CoveoDiagnosticPage.CreateStateVerifier()\n\n\n</code></pre> \n<p>Thanks,</p> \n<p>Rajkumar G</p>\n                </div>",
      "sitemaplastmodified" : 1443398400000,
      "collection" : "default",
      "syssourcetype" : "Sitemap",
      "indexeddate" : 1517942307000,
      "questionnumber" : 6162,
      "connectortype" : "SitemapCrawler",
      "filetype" : "html",
      "sysclickableuri" : "https://answers.coveo.com/questions/6162/error-in-coveo-diagnostic-page.html",
      "sysfiletype" : "html",
      "language" : [ "English" ],
      "sitemapparenturl" : "http://answers.coveo.com/questions/1.xml",
      "questionauthor" : "rajkumar",
      "sitename" : "Answers",
      "sysrowid" : 5447819,
      "uri" : "https://answers.coveo.com/questions/6162/error-in-coveo-diagnostic-page.html",
      "syscollection" : "default",
      "sitelanguage" : [ "English" ]
    },
    "Title" : "Error in Coveo Diagnostic Page",
    "Uri" : "http://answers.coveo.com/questions/6162/error-in-coveo-diagnostic-page.html",
    "PrintableUri" : "https://answers.coveo.com/questions/6162/error-in-coveo-diagnostic-page.html",
    "ClickUri" : "https://answers.coveo.com/questions/6162/error-in-coveo-diagnostic-page.html",
    "UniqueId" : "42.4247$http://answers.coveo.com/questions/6162/error-in-coveo-diagnostic-page.html",
    "Excerpt" : "As advised coveo, i have updated Coveo for Sitecore to 72 3.0 (1084). ... Now im getting below error in diagnostic page. ... Also im not able to do indexing due to this error. ... Please help me to...",
    "FirstSentences" : null
  }